apps/api/src/bootstrap.ts

```diff
@@ -23,6 +23,7 @@ const compression = require('compression');
 
 const extendedBodySizeRoutes = [
   '/v1/events',
+  '/v1/events/trigger',
```

Comment:
This line won't fix the user's problem:
1. The line isn't necessary because Next.js automatically accepts the prefix as the path. You can check it in the `Path examples` section here: https://expressjs.com/en/4x/api.html#app.use.
2. The `extendedBodySizeRoutes` array actually limits routes to 20 MB, rather than increasing the limit. We should probably rename it to make that clearer.

Reply:
You are right, the URL should have worked before. 
Could it be that there are also some issues with the storage provider implementation?
When I test it locally with the localstack container, it seems to work. But when I try to run it with Azure Blob Storage, the error occurs.